render_html_test.go

```diff
@@ -301,8 +347,9 @@ func TestHTMLNoRace(t *testing.T) {
 		Directory: "fixtures/basic",
 	})
 
+	var err error
```

Comment:
We need to remove error check. This is causing Travis break as it introduces a race condition.